src/frontend/src/lib/components/ui/MessageBox.svelte

```diff
@@ -1,18 +1,20 @@
 <script lang="ts">
 	import IconInfo from '$lib/components/icons/lucide/IconInfo.svelte';
 
-	export let level: 'info' | 'light-warning' = 'info';
+	export let level: 'info' | 'light-warning' | 'error' = 'info';
 </script>
 
 <div
 	class="mb-4 flex items-start gap-4 rounded-xl px-4 py-3 text-sm font-medium sm:text-base"
 	class:bg-brand-subtle-alt={level === 'info'}
 	class:bg-warning-subtle={level === 'light-warning'}
+	class:bg-pale-pink={level === 'error'}
```

Comment:
and here it should be `bg-error-subtle-alt` (double-check please)

Reply:
Yes, thanks!